src/backend/consumer/macos.rs

```diff
@@ -209,9 +210,14 @@ impl EventConsumer for MacOSConsumer {
                 PointerEvent::Frame { .. } => {}
             },
             Event::Keyboard(keyboard_event) => match keyboard_event {
-                KeyboardEvent::Key { .. } => {
-                    /*
-                    let code = CGKeyCode::from_le(key as u16);
+                KeyboardEvent::Key { time: _, key, state } => {
+                    let code = match KeyMap::from_key_mapping(KeyMapping::Evdev(key as u16)) {
+                        Ok(k) => CGKeyCode::from_le(k.mac),
```

Comment:
I'm unsure about the endianness here: Should this be `from_ne(k.mac)` maybe? I had that there more or less as a placeholder.

Reply:
`from_ne` doesn't appear to be defined. I tried `from_be` too, which gives odd results (which makes sense since `u16` is 2 bytes).

Reply:
Actually there is no conversion necessary at all. I believe I missed that it's just a typedef to u16. So this should be `k.mac as CGKeyCode` then.